src/routes/+error.svelte

```diff
@@ -12,16 +12,20 @@
 </svelte:head>
 
 <section title={$page.error?.message}>
-  <em>404</em>
-  <h1>Frank can't find where you're going!</h1>
-  <Button text={'Return to Home'} link={'/'} />
-  <img src="/assets/png/lost-frank.png" alt="404 - Page Not Found" />
+  <div class="container">
+    <div>
+      <em>404</em>
+      <h1>Frank can't find where you're going!</h1>
+      <Button text={'Return to Home'} link={'/'} />
+    </div>
+    <img src="/assets/png/lost-frank.png" alt="404 - Page Not Found" />
+  </div>
 </section>
 
 <Footer />
 
 <style>
-  section {
+  /* section {
```

Comment:
Nit: Delete unused comments.

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/815/commits/f1cb7fb2574a82a88b7e0f7038a0c15013fe9b91>.